person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+  attr_reader :id
```

Comment:
## Nice Job for using the shortcut Method 💯 
- Here is the link you can Read in your free time: [Useful shortcuts: attr_reader, attr_writer, and attr_accessor ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor)